core/components/molecules/tabs/tabs.js

```diff
@@ -92,6 +98,8 @@ class Tabs extends React.Component {
               onClick={() => this.changeTab(index)}
               key={index}
               selected={selectedIndex === index}
+              tabIndex="0"
```

Comment:
Hey 👋 What is this `tabIndex` for? All tabs would have it hard set to `"0"`.

Reply:
Hey!!
https://stackoverflow.com/questions/4115953/where-do-tabindex-0-html-elements-end-up-in-the-tabbing-order
and https://www.w3.org/TR/html401/interact/forms.html#tabbing-navigation

`Elements that have identical tabindex values should be navigated in the order they appear in the character stream.`

Reply:
Excellent! Thank you for the clarification.